### PR TITLE
Fixes #23806 - explicitly use Rails 5.2 compatible gems

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
Depends on

- [x] https://github.com/theforeman/foreman-packaging/pull/2595
- [x] https://github.com/theforeman/foreman-packaging/pull/2596
- [x] https://github.com/theforeman/foreman-packaging/pull/2597

**Comment by mmoll** [comment]
test failures unrelated :green_heart:

File: Gemfile [hunk 1]
```diff
@@ -13,17 +13,17 @@ else
 end
 
 gem 'rest-client', '>= 2.0.0', '< 3', :require => 'rest_client'
-gem 'audited', '~> 4.7'
+gem 'audited', '~> 4.7', '>= 4.7.1'
```
Comment (ekohl) [inline Gemfile]: The script I use to convert this to the spec file can't handle this well (it generates 3 Require statements). I'd prefer `'>= 4.7.1', '< 5'`.

Comment (mmoll) [reply Gemfile]: ok, changed :+1:

**Review by ekohl — APPROVED** [review]

**Review by tbrisker — APPROVED** [review]

**Comment by tbrisker** [comment]
Thanks @mmoll !

**Pull request merged**